ts/private/ts_proto_library.bzl

```diff
@@ -96,8 +96,9 @@ ts_proto_library = rule(
     implementation = _ts_proto_library_impl,
     attrs = dict({
         "gen_connect_es": attr.bool(
-            doc = "whether to generate service stubs with gen-connect-es",
-            default = True,
+            doc = """whether to generate service stubs with gen-connect-es
+            Deprecated: no longer needed, see https://github.com/connectrpc/connect-es/blob/main/MIGRATING.md""",
+            default = False,
```

Comment:
Is this not a breaking change?

Reply:
It is, but this whole module is not semver:
https://github.com/aspect-build/rules_ts/blob/de8d8b99cc2a4bdba9378956ec4bdf6ffeb9a9e6/ts/proto.bzl#L1-L6